src/node_crypto.cc

```diff
@@ -3185,7 +3185,7 @@ static ManagedEVPPKey GetPublicOrPrivateKeyFromJs(
           is_public = false;
           break;
         default:
-          CHECK(!"Invalid key encoding type");
+          CHECK(false);
```

Comment:
We have a macro for this: `UNREACHABLE()`

Perhaps it could be extended to include an abort message.